slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -22,9 +25,43 @@ using viam::common::v1::PoseInFrame;
 #define MAX_COLOR_VALUE 255
 const std::string strRGB = "/rgb";
 const std::string strDepth = "/depth";
+const auto HEADERTEMPLATE =
+    "VERSION .7\n"
+    "FIELDS x y z\n"
+    // NOTE: If a float is more than 4 bytes
+    // on a given platform
+    // this size will be inaccurate
+    "SIZE 4 4 4\n"
+    "TYPE F F F\n"
+    "COUNT 1 1 1\n"
+    "WIDTH %d\n"
+    "HEIGHT 1\n"
+    "VIEWPOINT 0 0 0 1 0 0 0\n"
+    "POINTS %d\n"
+    "DATA binary\n";
 
 namespace viam {
 
+/*
+applies the mapSize to the HEADERTEMPLATE
+returning the pcd header as a string
+*/
```

Comment:
[opt]we haven't been using block comments for other functions, not something that needs to change though